cargo-audit/src/presenter.rs

```diff
@@ -323,6 +323,14 @@ impl Presenter {
         } else if let Some(url) = &metadata.url {
             self.print_attr(color, "URL:      ", url);
         }
+
+        if let Some(cvss) = &metadata.cvss {
+            self.print_attr(
+                color,
+                "Severity: ",
+                format!("{}, {}", cvss.score().value(), cvss.score().severity()),
```

Comment:
A bit late here but I might've recommended

```rust
format!("{} ({})", cvss.score().value(), cvss.score().severity()),
```

Reply:
Yes, that'd be great